src/javascript/EditPanel/header/ContentBreadcrumb/ContentPath/ContentPath.container.jsx

```diff
@@ -23,10 +25,20 @@ const ContentPathContainer = ({path}) => {
     });
 
     const handleNavigation = path => {
+        let newMode = 'pages';
+
+        if (path.startsWith(`/sites/${site}/files`)) {
```

Comment:
you miss the trailing `/` .. if a page is named `filesPage` at root level, it will be detected as a media